### PR TITLE
added instructions on how to do an editable installation

### PR DESCRIPTION
**Pull request by elenajp**
Hi,

I added instructions on how to do an editable installation. Please let me know if you would like me to add more details. I was thinking of writing explanations of what each command does but I was unsure if that was necessary. Thanks for this opportunity, it feels great to be able to contribute to a real project for the first time. I hope what I did was ok. 

Elena :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,8 +74,19 @@ either column and find a ticket you like. Create a branch from the HEAD of
...
+
+The editable installation mode implies that the source code of PursuedPyBear being installed is available in a local directory. Once PursuedPyBear is installed in editable mode, users expect that changes to the PursuedPyBear python code in the local source tree become effective without the need of a new installation step. To do an editable installation:
+
+    mkdir ppb
+    python3 -m venv ppb-venv
+    source ppb-venv/bin/activate
+    cd ppb-venv
+    git clone https://github.com/ppb/pursuedpybear.git
+    cd pursuedpybear
+    pip install --upgrade -r requirements.txt
```
Comment (pathunstrom) [inline CONTRIBUTING.md]: This should reference `requirements-test.txt` instead. The `requirements.txt` is fed into the setup configuration so you get for free as part of the pip install.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
 
-
 ## Maintainers
 
-Name | Email | Twitter |
------|-------|---------|
-[Piper Thunstrom](https://github.com/pathunstrom) | [pathunstrom@gmail.com](mailto:pathunstrom@gmail.com) | [pathunstrom](https://twitter.com/pathunstrom)
-[Luna Lunapiena](https://github.com/lunacodes) | [lunacodes@gmail.com](lunacodes@gmail.com) | [lunacodess](https://twitter.com/lunacodess)
-[Jamie Bliss](https://github.com/AstraLuma/) | [jamie@ivyleav.es](jamie@ivyleav.es) | [AstraLuma](https://twitter.com/AstraLuma)
+| Name                                              | Email                                                 | Twitter                                        |
```
Comment (pathunstrom) [inline CONTRIBUTORS.md]: This is a lot of formatting changes, can we revert this file and just add your information?

Trying to keep the nice text format will be a losing game over time regardless of how large we make it.

**Review by pathunstrom — CHANGES_REQUESTED** [review]
Instructions look good, a few comments to make sure the environment is correct.

Also would like the contributors.md formatting removed just because keeping plain text tables with shared widths is a losing proposition in a community project like this.

Thanks!

**Comment by pathunstrom** [comment]
And while we work through the review, I saw you included a twitter account, when we merge, would you like a shoutout from the official twitter?

File: CONTRIBUTING.md (hunk 1 above)
Comment (elenajp) [reply CONTRIBUTING.md]: ok great, I will make the change. Thanks!

File: CONTRIBUTORS.md (hunk 2 above)
Comment (elenajp) [reply CONTRIBUTORS.md]: I realised that one of my extensions in vscode was causing the formatting upon saving. I disabled it and will commit again. So the nice format is back :)

**Comment by elenajp** [comment]
> And while we work through the review, I saw you included a twitter account, when we merge, would you like a shoutout from the official twitter?

Yeah would would be awesome, thank you 😀

**Comment by elenajp** [comment]
So the changes have been made. This is my first real contribution to open source so I am getting used to the flow of things on github so I hope everything is ok now. It makes me feel great to be able to contribute! Let me know if I need to make any further changes. Thanks!

**Review by pathunstrom — APPROVED** [review]
Thanks for fixing the formatting thing, I'm sorry our tool chain required that work!

bors r+

**Comment by elenajp** [comment]
I had fun working on this issue and got to learn more about open source. Thank you! 😀

**Comment by pathunstrom** [comment]
I'm glad! Feel free to grab other tickets whenever you like!

**Pull request merged**